app/controllers/users/sessions_controller.rb

```diff
@@ -31,10 +31,9 @@ def active
     end
 
     def timeout
-      if sign_out
-        analytics.track_event(Analytics::SESSION_TIMED_OUT)
-        flash[:timeout] = t('session_timedout')
-      end
+      analytics.track_event(Analytics::SESSION_TIMED_OUT)
+      sign_out
```

Comment:
Yep. I can't think of a scenario where there wouldn't be a user to sign out when we make the call to `sign_out`.